src/server/cmd/pachctl/cmd/cmd.go

```diff
@@ -2,41 +2,57 @@ package cmd
 
 import (
 	"bufio"
+	"errors"
 	"fmt"
 	"io"
+	"io/ioutil"
+	"log"
 	"os"
 	"text/tabwriter"
 	"time"
 
 	"google.golang.org/grpc"
+	"google.golang.org/grpc/grpclog"
 
 	"github.com/pachyderm/pachyderm/src/client"
 	"github.com/pachyderm/pachyderm/src/client/version"
 	pfscmds "github.com/pachyderm/pachyderm/src/server/pfs/cmds"
 	ppscmds "github.com/pachyderm/pachyderm/src/server/pps/cmds"
 	"github.com/spf13/cobra"
+	"go.pedge.io/lion"
 	"go.pedge.io/pb/go/google/protobuf"
 	"go.pedge.io/pkg/cobra"
 	"go.pedge.io/proto/version"
 	"golang.org/x/net/context"
 )
 
 func PachctlCmd(address string) (*cobra.Command, error) {
+	var verbose bool
 	rootCmd := &cobra.Command{
 		Use: os.Args[0],
 		Long: `Access the Pachyderm API.
 
 Envronment variables:
   ADDRESS=0.0.0.0:30650, the server to connect to.
 `,
+		PersistentPreRun: func(cmd *cobra.Command, args []string) {
+			if !verbose {
+				// Silence any grpc logs
+				grpclog.SetLogger(log.New(ioutil.Discard, "", 0))
+				// Silence our FUSE logs
+				lion.SetLevel(lion.LevelNone)
```

Comment:
Yes, this will silence the annoying 'file not found' errors that would popup from normal usage via fuse mount.

The `PersistentPreRun` makes sure that this function runs before any of the child command's `Run` callbacks are called. So - we make sure we set the log levels before the 'real' code runs.